### DISCUSSION TITLE
Yi 1080p Home 9FCN hacked successfully.

### DISCUSSION THREAD

**Discussion by lxdfly** [roleoroleo/yi-hack-Allwinner#279] [Show and tell]
I bought a Yi 1080p Home  with  9FCN prefix DID  in China and I hacked it successfully. Thanks to roleoroleo.

**Comment by roleoroleo** [comment]
Thank you for your feedback.
Could you please check the fw version in the yi app?

**Reply by lxdfly** [reply]
the firmversion is 8.2.0.0A_202012041528

**Reply by lxdfly** [reply]
I found the snapshot picture  has two timestamp,one is at the  top left corner ,and another is at the  bottom right corner ,but those two timestamp  are  different. The bottom right corner stamp is right.
![timestamp](https://user-images.githubusercontent.com/22286259/114141414-21deaf80-9944-11eb-9478-99d1bb2a6e6b.png)

**Reply by lxdfly** [reply]
And the recorded video only has the worng stamp at the  top left corner

**Comment by roleoroleo** [comment]
The watermark on the bottom-right is added from the hack.
The wrong timestamp depends on the timezone settings. If you change timezone, yi firmware writes the wrong time.

**Reply by lxdfly** [reply]
Thank you . I got it . I'm in China (CST8).   Now I can set the TimeZone to GMT0 on configuration page so  that I can get the correct time on video recorded( top-left corner)  .
